environment.yml

```diff
@@ -56,3 +56,4 @@ dependencies:
 - edflib-python
 - pybv
 - mamba
+- mne-gui-addons
```

Comment:
To keep moving forward let's omit this. We might need to add some `@pytest.filterwarnings(...)` to some tests to get it to be green since it should use the deprecated code for now. I'll push a couple quick commits then we can merge and add this back in once the conda-forge people get back to us on the `vtk` issue...
```suggestion
```